src/train.jl

```diff
@@ -3,7 +3,12 @@ const DEFAULT_DATA = Iterators.cycle((NullData(),))
 Base.iterate(::NullData, i=1) = nothing
 Base.length(::NullData) = 0
 
-@deprecate sciml_train(loss, θ, opt, args...; kwargs...) GalacticOptim.solve(optprob, opt)
+function sciml_train(loss, θ, opt, adtype; kwargs...)
+  @warn("sciml_train has been deprecated in favor of GalacticOptim.jl (https://github.com/SciML/GalacticOptim.jl)") 

```

Comment:
Should this warning be added to all the `sciml_train` calls?

Reply:
@ChrisRackauckas?

Reply:
yeah that would be good.